ubuntu-personal-store/ubuntu-personal-store.qml.in

```diff
@@ -49,6 +49,11 @@ ApplicationWindow {
     id: webView
     focus: true
 
+    context: WebContext {
+        dataPath: Qt.application.arguments[1]
```

Comment:
Good point. I've made the dataPath argument more explicit by expecting "CookiePath=xxx"